handlers/istio_config.go

```diff
@@ -331,7 +202,6 @@ func IstioConfigCreate(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 	namespace := params["namespace"]
 	objectType := params["object_type"]
-	objectSubtype := params["object_subtype"]
```

Comment:
I can see a bunch of remaining `object_subtype` in code, mostly related to swagger / doc, I guess they should go away too?
(Just searched for "object_subtype" in all files)

Reply:
Ops, you are right, those should be cleaned as well.